src/pages/TransactionDetails/components/EventsList/EventExtraData.tsx

```diff
@@ -0,0 +1,28 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import { DataDecode, DecodeMethodEnum } from 'components';
+
+interface EventExtraDataType {
```

Comment:
As per discussed, this should instead be `EventExtraDataUITtype`, for consistency. 👍

Reply:
updated

Reply:
Great. 🔥